common/src/main/java/tachyon/util/io/FileUtils.java

```diff
@@ -121,20 +121,12 @@ public static void setLocalFileStickyBit(String file) {
    * @throws java.io.IOException
    */
   public static void createBlockPath(String path) throws IOException {
-    File localFolder;
     try {
-      localFolder = new File(PathUtils.getParent(path));
+      createStorageDirPath(PathUtils.getParent(path));
```

Comment:
The previous code did not call setLocalFileStickyBit(), and now it does. Is that intentional and what are the implications?